os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -74,6 +77,8 @@ struct rtl8730e_lcdc_info_s g_rtl8730e_config_dev_s = {
 		.lcd_layer_enable = rtl8730e_lcd_layer_enable,
 		.lcd_put_area = rtl8730e_lcd_put_area,
 		.backlight = rtl8730e_control_backlight,
+		.power_off = rtl8730e_power_off,
+		.power_on = rtl8730e_power_on
```

Comment:
maybe lcd power off and on. Let's add `lcd`

Reply:
Updated